docs/azure-ephemeral-disks.md

```diff
@@ -0,0 +1,23 @@
+Using ephemeral storage within clusters deployed by Muchos for Azure
+--------------------------------------------------------------------
+
+By default for Azure based clusters, Muchos will create 3 data disks, each of size 128GiB, attached to each VM. This provides
+persistent storage which ensures that the data in HDFS is safe and consistent even if the VMs are deallocated (stopped).
+However, certain cases, in case you'd like to use only the ephemeral / temporary disk storage for HDFS, you first need to 
```

Comment:
```suggestion
However, if you'd like to use only the ephemeral / temporary disk storage for HDFS, you first need to 
```

Reply:
Thanks @keith-turner for catching the grammatical issue and proposing the edit.